src/chainstate/stacks/boot/costs.clar

```diff
@@ -0,0 +1,495 @@
+;; the .costs contract
+
+;; Helper Functions
```

Comment:
Overall, I think for this contract it would be helpful to add a comment to each cost function to briefly explain what the instance characteristic variables represent (i.e. what's _n_ represent?).

Reply:
https://github.com/blockstack/stacks-blockchain/issues/2061